dotcom-rendering/src/web/components/Figure.tsx

```diff
@@ -175,13 +175,19 @@ export const Figure = ({
 			</figure>
 		);
 	}
+
+	let spacefinderComponent: 'rich-link' | 'numbered-list-title' | null = null;
+	if (role === 'richLink') {
+		spacefinderComponent = 'rich-link';
+	} else if (isNumberedListTitle) {
+		spacefinderComponent = 'numbered-list-title';
+	}
```

Comment:
In the `else` condition should we add `data-spacefinder-component` as an attribute?

Reply:
Don't quite follow, can you clarify what you mean?

Reply:
Sure - so we currently have `if (x) else if (y)`.

I was wondering what happens (and perhaps what should happen) in the unhandled `else` condition.

But I can see now that the value of `data-spacefinder-component` will be `null` so the attribute won't actually be added to the dom!

Sorry please proceed :)

Reply:
Ahhhh gotcha. Agree the intent isn't super clear but hopefully adding to the docs will help :)